node/sc/bridgepool/sorted_map_list.go

```diff
@@ -203,6 +220,27 @@ func (m *ItemSortedMap) Remove(nonce uint64) bool {
 	return true
 }
 
+// Forward removes all items from the map with a nonce lower than the
+// provided threshold. Every removed transaction is returned for any post-removal
+// maintenance.
+func (m *ItemSortedMap) Forward(threshold uint64) items {
```

Comment:
How about this?
```suggestion
func (m *ItemSortedMap) Truncate(threshold uint64) items {
```

Reply:
This name is from `txSortedMap`. So I think original name is better.